js/interface.js

```diff
@@ -903,6 +928,112 @@ function showNothingFoundAlert(isShow) {
   }
 }
 
+function showAlertWithButtonGoToFolder(item, area) {
+  var goToButton = $('#alert-btn-action').attr('data-type', area.type);
+
+  if (area.type == 'organization') {
+    goToButton.attr('data-id', area.orgId);
+  } else if (area.type == 'app') {
+    goToButton.attr('data-id', area.appId);
+  } else {
+    goToButton.attr('data-id', area.id);
+  }
+
+  $('.alert-action').addClass('active');
+  $('.alert-message').text(item.length + ' item(s) moved');
+  setTimeout(function() {
+    setAlertVisibilityWhenMovingItems(false);
+  }, $goToFolderAlertTimeout);
```

Comment:
the `$` prefix is used for jquery variables, this should be an integer instead

Reply:
fixed